### DISCUSSION TITLE
Nested Workspace Todo's

### DISCUSSION THREAD

**Discussion by ZahlenZ** [dooit-org/dooit#194] [Q&A]
Just started with dooit this morning, love it so far. Curious if there is a way to show all the todos from nested workspaces in the parent? For instance I have my workspace for college, with children for each class. It would be nice if I could see the todos for each of the classes when the parent workspace is selected, is this possible? Thanks!

**Comment by kraanzu** [comment]
Hi! No, Not as of the current version. I'm working on a newer version which will allow this. I was hoping to complete it by this week but probably it'll take one more

I'd also love to know how do you imagine the workflow to be? Can you write some kind of steps of what you expect to happen. Go all out

**Reply by ZahlenZ** [reply]
wow, thanks for the quick reply, good to know!
